AirSkyBoat/AirSkyBoat
# Moves item.lua and to enum and removes more keyitems

**_I affirm:_**


## Please enter a player-facing description
Nothing to say here


## What does this pull request do? (Please be technical)
Moves item.lua and to enum


## Steps to test these changes
Nothing - use an item/skill or anything that had require for items.lua and see that it still works


## Special Deployment Considerations
Nothing to say here
